### PR TITLE
Mg remove unstyled

### PR DESCRIPTION
**Pull request by Magnus-G**
image

### CODE REVIEW

File: source/stylesheets/_refills-header.scss [hunk 1]
```diff
@@ -88,14 +88,25 @@
...
 
     @include media($refills-large-screen) {
       font-size: 0.8em;
     }
   }
+
+  .empties-banner {
+    background: #FCFFE0;
+    border-radius: 2em;
+    box-shadow: 0 2px 0px #E3E8AA;
```
Comment (brendastorer) [inline source/stylesheets/_refills-header.scss]: If we are going with purple for Empties, what do you think of making the banner purple? #776FAA

**Comment by brendastorer** [comment]
Probably a good idea from the :dog: to move that color into a variable.

File: source/stylesheets/_refills-header.scss [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
     margin-bottom: 4em;
-    padding: 6em 0 10em;
+    padding: 6em 0 8em;
   }
 
   @include media($refills-large-screen) {
     border-bottom: 2px solid darken($refills-hero-background, 5%);
     margin-bottom: 4em;
-    padding: 6em 0 11em 0;
+    padding: 6em 0 8em 0;
```
Comment (tysongach) [inline source/stylesheets/_refills-header.scss]: +1

File: source/stylesheets/_refills-header.scss [hunk 3]
```diff
@@ -88,14 +88,25 @@
...
+    margin-bottom: 4em;
     padding: 0.5em 1.4em;
 
     @include media($refills-large-screen) {
       font-size: 0.8em;
     }
   }
+
+  .empties-banner {
+    background: #FCFFE0;
```
Comment (tysongach) [inline source/stylesheets/_refills-header.scss]: +1, also, prefer use of `background-color` here instead of just `background` per our Guides.

**Pull request closed without merging**